contracts/SortitionPool.sol

```diff
@@ -17,31 +17,72 @@ contract SortitionPool is SortitionTree, Ownable {
   using DynamicArray for DynamicArray.UintArray;
   using DynamicArray for DynamicArray.AddressArray;
 
-  struct PoolParams {
-    IStaking stakingContract;
-    uint256 minimumStake;
-    uint256 poolWeightDivisor;
+  /// @notice Governance delay that needs to pass before any risk manager
+  ///         parameter change initiated by the governance takes effect.
+  uint256 public constant GOVERNANCE_DELAY = 48 hours;
```

Comment:
Let's do it without governable delay. If we need a governable delay, we will add it in `RandomBeacon` or application controlling the pool.

Reply:
Ok, done: https://github.com/keep-network/sortition-pools/pull/132/commits/de170130305d56bc9218fb811a46a9e09e1e646d